README.md

```diff
@@ -62,66 +62,97 @@ http localhost:8080/api/loadClients
 curl localhost:8080/api/loadClients
  
 ```
- 
+
 Stop RelayRunner by typing Ctrl-C or the stop button if run via F5
- 
+
 ## Client
- 
+
 ### Client Usage
- 
+
 #### Available Scripts
- 
+
 In the project directory, you can run:
- 
+
 - `npm clean-install`: Installs npm dependecies.
- 
+
 - `npm start` : Runs the app in development mode
- 
+
 - `npm run lint` : Runs Linter and automatically fixes Linter recommended changes
- 
+
 - `npm test` : Launches the test runner, jest, in the interactive watch mode.
- 
+
 - `npm run build` : Builds the app for production to the `build` folder.
- 
+
 - `npm run eject` : Removes the single build dependency from project and copies all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them.
   - *Note: this is a one-way operation. Once you `eject`, you can’t go back!*
- 
+
 ### Run the Client UI
- 
+
 #### Using bash shell for Client
- 
+
 > This will work from a terminal in Visual Studio Codespaces as well
- 
+
 1. Clone the Repo
       `git clone https://github.com/retaildevcrews/relayrunner.git`
- 
+
 2. Change into the client directory
       `cd client`
- 
+
 3. Install node dependencies
       `npm clean-install`
- 
+
 4. Run the client UI
       `npm start`
- 
+
 #### Testing the Client
- 
+
 Run tests on Client components and functions using `npm test`
- 
+
+## CosmosDB Collections
+
+### clientStatus
+
+Conveys the current status, time of that status, and the associated LoadClient's initial start-up configuration.
+
+- The TTL is set to 300 seconds
+
+```json
+{
+    "id": "5",
+    "partitionKey": "ClientStatus",
+    "entityType": "ClientStatus",
+    "lastUpdated": "2021-08-17T14:36:37.0897032Z",

```

Comment:
for DateTime values e.g LastUpdated and StartTime , are we going to use UTC time? is that right ?